src/flags.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+// -------------------------------------------------------------------------------
+// No requires or imports since this is loaded early in the cli lifecycle and we
+// want to minimize the number of packages that load before enabling require
+// instrumentation.
+// -------------------------------------------------------------------------------
+
+export interface ProcessLike {
+  argv: string[];
+  env: { [key: string]: string | undefined };
+}
+
+export function preprocessCliFlags(process: ProcessLike): void {
+  process.argv.map((arg) => {
+    if (arg === '--dev-debug') {
+      let debug = '*';
+      const filterIndex = process.argv.indexOf('--dev-filter');
```

Comment:
I don't like that name.  Is it a `--log-filter` or a `--debug-filter`.  It's not filtering devs.